### PR TITLE
FIX: Fix packaging

### LINKED ISSUE #878: point_collection (no key in -> program.vert['transform'] )
When I try to run the example point_collection 
https://github.com/vispy/vispy/blob/master/examples/collections/point_collection.py
with the ``mode='raw' or 'agg'``
the following error occures: 

```
KeyError                                  Traceback (most recent call last)
<ipython-input-11-7efb2e73af81> in <module>()
     15 panzoom = PanZoomTransform(canvas)
     16 
---> 17 points = PointCollection("raw", color="shared")
     18 points.append(np.random.normal(0.0, 0.5, (10000, 3)), itemsize=5000)
     19 points["color"] = (1, 0, 0, 1), (0, 0, 1, 1)

/opt/python3.4env/lib/python3.4/site-packages/vispy-0.4.0_dev-py3.4.egg/vispy/visuals/collections/point_collection.py in PointCollection(mode, *args, **kwargs)
     17 
     18     if mode == "raw":
---> 19         return RawPointCollection(*args, **kwargs)
     20     return AggPointCollection(*args, **kwargs)

/opt/python3.4env/lib/python3.4/site-packages/vispy-0.4.0_dev-py3.4.egg/vispy/visuals/collections/raw_point_collection.py in __init__(self, user_dtype, transform, vertex, fragment, **kwargs)
     71         # Set hooks if necessary
     72         program = self._programs[0]
---> 73         program.vert['transform'] = self.transform
     74 
     75     def append(self, P, itemsize=None, **kwargs):

/opt/python3.4env/lib/python3.4/site-packages/vispy-0.4.0_dev-py3.4.egg/vispy/visuals/shaders/function.py in __setitem__(self, key, val)
    222                 storage = self._expressions
    223             else:
--> 224                 raise KeyError('Invalid template variable %r' % key)
    225         else:
    226             raise TypeError('In `function[key]` key must be a string or '

KeyError: "Invalid template variable 'transform'"
```

### PR DESCRIPTION
**Pull request by Eric89GXL**
Closes #878. I have verified that on `master` if I use `python setup.py install` I have the same issues as @gabyx @brickgao and @bollu, and with this PR they are fixed.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Ready for review/merge from my end.

File: vispy/visuals/collections/util.py [hunk 1]
```diff
@@ -122,7 +122,10 @@ def fetchcode(utype, prefix=""):
     int i = index_x;
     vec4 _uniform;\n"""
 
-    _utype = {name: count for name, count, _ in _utype}
+    orig = _utype
+    _utype = dict()
+    for name, count, _ in orig:
+        _utype[name] = count
```
Comment (campagnola) [inline vispy/visuals/collections/util.py]: `_utype = dict([(name, count) for name, count, _ in _utype])`

**Comment by campagnola** [comment]
Happy to merge this with or without my suggestion.

File: vispy/visuals/collections/util.py (hunk 1 above)
Comment (Eric89GXL) [inline vispy/visuals/collections/util.py]: ahh, wasn't sure if there was a Py26-friendly way to do it, but this is probably it

**Comment by campagnola** [comment]
Grazie!

**Pull request merged**